framework_tvb/tvb/core/entities/file/files_helper.py

```diff
@@ -108,8 +114,6 @@ def rename_project_structure(self, project_name, new_name):
 
             if os.path.exists(new_full_name):
                 raise IOError("Path exists %s " % new_full_name)
-
-            os.rename(path, new_full_name)
```

Comment:
Is this removal by intention ?
What if used in a deployment where encryption is disabled ?

Reply:
You are right. This line was accidentally deleted.